Public/styles/about.css

```diff
@@ -0,0 +1,64 @@
+.organizer .image {
+    position: relative;
+}
+.organizer .image .social-media {
+    align-items: center;
+    justify-content: space-around;
+    display: none;
+    position: absolute;
+    background-color: #0C1D28;
+    height: 100%;
+    width: 100%;
+    top: 0;
+}
+.organizer .image:hover .social-media {
+    display: flex;
+}
+.organizer .image:hover .social-media a {
+    padding: 5px;
+    height: 40px;
+    width: 40px;
+}
+
+.organizer .image img {
+    width: 100%;
+}
+
+.organizer .image {
+    border-radius: 3px;
+    overflow: hidden;
+}
+
+.organizer .image.avatar {
+    background-color: #0b1e28;
+}
+
+.organizer .card {
+    margin: 15px 0;
+    background: #0C1D28;
+    border-radius: 5px;
+    padding: 10px;
+    text-align: center;
+
+    border-left: 2px solid #F88A36;
+    border-right: 2px solid #FD2020;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    background-position: 0 0, 0 100%;
+    background-repeat: no-repeat;
+    -webkit-background-size: 100% 2px;
+    -moz-background-size: 100% 2px;
+    background-size: 100% 2px;
+    background-image: -webkit-linear-gradient(left, #F88A36 0%, #FD2020 100%), -webkit-linear-gradient(left, #F88A36 0%, #FD2020 100%);
+    background-image: -moz-linear-gradient(left, #F88A36 0%, #FD2020 100%), -moz-linear-gradient(left, #F88A36 0%, #FD2020 100%);
+    background-image: -o-linear-gradient(left, #F88A36 0%, #FD2020 100%), -o-linear-gradient(left, #F88A36 0%, #FD2020 100%);
+    background-image: linear-gradient(to right, #F88A36 0%, #FD2020 100%), linear-gradient(to right, #F88A36 0%, #FD2020 100%);
+}
+.organizer .card .title {
+    font-size: 22px;
+}
+
+.organizer {
+    padding: 50px;
```

Comment:
I'd suggest to change it to `padding: 0 50px;` to get the spacing between headers and content consistent 😊